website/src/pages/documents/document.page.tsx

```diff
@@ -82,6 +83,10 @@ const TabSet = ({ doc }: { doc: Document }) => {
           </Tab>
         </TabList>
       </div>
+      <div id="scroll-top" className={css.scrollTop} onClick={() => window.scrollTo({top: 0, behavior: 'smooth'})}>
```

Comment:
Maybe consider using an anchor tag somewhere in the scroll to top button so that hovering over the element indicates someone can interact with it. This also helps browsers and screen readers understand what that element does. On that note, this element seems like it is not easily accessible through non-pointer navigation. Maybe consider using tabbable or sim.?

Reply:
You should definitely have some text like "scroll to top" alongside the arrow. Adding a text label makes the visual semantics more clear. As it stands now, the function of the arrow is unclear. On mobile (and maybe even on desktop), adding just "Top" or "To top" should be enough. We could ask Ellen about using ᎤᏍᎪᎵ for 'top, peak' in addition.